arm9/source/game/ticket.c

```diff
@@ -120,7 +120,7 @@ u32 BuildVariableFakeTicket(Ticket** ticket, u32* ticket_size, const u8* title_i
 }
 
 u32 BuildFakeTicket(Ticket* ticket, const u8* title_id) {
-    Ticket* tik;
+    Ticket* tik = NULL;
```

Comment:
```suggestion
    Ticket tik;
```

Reply:
I believe this has to stay as `Ticket*` due to how it is passed to BuildVariableFakeTicket on line 125

Reply:
> I believe this has to stay as `Ticket*` due to how it is passed to BuildVariableFakeTicket on line 125

Checked again signature of `BuildVariableFakeTicket`. You're right.